fhi_users/auth/auth_utils.py

```
@@ -64,17 +64,22 @@ def validate_username(username: str) -> bool:
+    Validate password using Django's built-in password validators.
+
+    Returns True if the password passes all validators, False otherwise.
+    This provides stronger security than basic length/digit checks by also
+    checking against common passwords and user attribute similarity.
+    """
+    from django.contrib.auth.password_validation import validate_password as django_validate
+    from django.core.exceptions import ValidationError as DjangoValidationError
+
+    try:
+        django_validate(password, user=user)
+        return True
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)